src/OpenTelemetry/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ Notes](../../RELEASENOTES.md).
 
 ## Unreleased
 
+* The experimental APIs previously covered by `OTEL1003`
+  (`MetricStreamConfiguration.CardinalityLimit`) will now be part of the public
```

Comment:
will be -> is?

Reply:
I wonder how folks think about https://www.theregister.com/2024/10/08/linus_torvalds_grammar_complaint/?

Reply:
This wording is also used in other experimental APIs in this CHANGELOG.

Reply:
I think "is" is better. Would be good to follow up and make changes everywhere. Not a blocker for this.

Reply:
Created a PR #5940 for this.